lib/glific/whatsapp_forms.ex

```diff
@@ -71,14 +71,26 @@ defmodule Glific.WhatsappForms do
   Deactivates a WhatsApp form by its Meta Flow ID.
   """
   @spec deactivate_whatsapp_form(non_neg_integer()) ::
-          {:ok, WhatsappForm.t()} | {:error, String.t()}
+          {:ok, %{whatsapp_form: WhatsappForm.t()}} | {:error, String.t()}
   def deactivate_whatsapp_form(id) do
     with {:ok, form} <- get_whatsapp_form_by_id(id),
          {:ok, updated_form} <- update_form_status(form, :inactive) do
       {:ok, %{whatsapp_form: updated_form}}
     end
   end
 
+  @doc """
+  activate a WhatsApp form by its Meta Flow ID.
```

Comment:
```suggestion
  Activate a WhatsApp form by its Meta Flow ID.
```

Lets also mention whats the difference between activate and publish here